### PR TITLE
Extend visualization frame tests

### PR DESCRIPTION
**Pull request by tarzzz**

### CODE REVIEW

**Comment by tarzzz** [comment]
@moorepants , I have extended tests to include testing for numerical methods as well

File: pydy_viz/tests/test_visualization_frame_scene.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from sympy.physics.mechanics import *
```
Comment (moorepants) [inline pydy_viz/tests/test_visualization_frame_scene.py]: Avoid `import *`. Always have explicit imports or import the whole module.

File: pydy_viz/tests/test_visualization_frame_scene.py [hunk 2]
```diff
@@ -24,41 +23,45 @@ def __init__(self):
         self.point_list1 = [[2, 3, 1], [4, 6, 2], [5, 3, 1], [5, 3, 6]]
         self.point_list2 = [[3, 1, 4], [3, 8, 2], [2, 1, 6], [2, 1, 1]]
 
-        self.mesh_shape1 = MeshShape('mesh_shape1', \
-                                points=self.point_list1, color='blue')    
-        self.mesh_shape2 = MeshShape('mesh_shape2', \
-                                points=self.point_list2, color='red')
```
Comment (moorepants) [inline pydy_viz/tests/test_visualization_frame_scene.py:30]: I'm fine if you remove the Meshes from this test. It doesn't really add anything to the VizFrame tests. Just test the Mesh stuff in test_shape.py.

File: pydy_viz/tests/test_visualization_frame_scene.py [hunk 3]
```diff
@@ -24,41 +23,45 @@ def __init__(self):
...
+        #TODO replace by a mesh shape ideally, when its implemented
+        self.mesh_shape1 = Cylinder()
+        self.mesh_shape2 = Cylinder()
                                
                                
         self.Ixx, self.Iyy, self.Izz = symbols('Ixx Iyy Izz')
         self.mass = symbols('mass')
-        self.parameters = [self.Ixx, self.Iyy, self.Ixzz, self.mass]
+        self.parameters = [self.Ixx, self.Iyy, self.Izz, self.mass]
         self.param_vals = [0, 0, 0, 0]
```
Comment (moorepants) [inline pydy_viz/tests/test_visualization_frame_scene.py:43]: It is typically best to provide numbers in tests. If you test with zeros everywhere there are often an infinite number of ways to get zeros in the outputs, which does you no good.

File: pydy_viz/tests/test_visualization_frame_scene.py [hunk 4]
```diff
@@ -24,41 +23,45 @@ def __init__(self):
...
-                    [sin(self.p[1]), -sin(self.p[0])*cos(self.p[1]), \
-                                   cos(self.p[0])*cos(self.p[1]),10], \
-                                     [0, 0, 0, 1]]
+        
+        #To make it more readable      
+        p = self.p
+        q = self.q 
+        #Here is the dragon ..
+        self.transformation_matrix = \
+            [[cos(p[1])*cos(p[2]), sin(p[2])*cos(p[1]), -sin(p[1]), 0], \
```
Comment (moorepants) [inline pydy_viz/tests/test_visualization_frame_scene.py:63]: Backslashes are necessary when you have parentheses/brackets opening a line.

**Comment by moorepants** [comment]
Why doesn't this PR have all of the commits for visualization frame tests?

Be sure to make your PR's atomic in the future. Right now it is difficult to comment on the whole of vframe tests when the other commits for it have already been merged or are in another branch.

**Comment by moorepants** [comment]
Also, I'm not expecting you to have the source and tests for shapes and vframes **merged** by Monday. Complete pull requests are ideal. I see you've been opening PRs and merging them before anyone has a chance to look at them. There is no reason to do that.

File: pydy_viz/tests/test_visualization_frame_scene.py (hunk 2 above)
Comment (tarzzz) [inline pydy_viz/tests/test_visualization_frame_scene.py:30]: I need any shape here, for initializing vframes, I will keep a cylinder shape only.

**Pull request merged**